demo/requirements.txt

```diff
@@ -1,2 +1,2 @@
--e git+https://github.com/mindee/doctr.git#egg=python-doctr[tf]
+-e git+https://github.com/mindee/doctr.git#egg=python-doctr[all]
```

Comment:
So that part is tricky: all means that it will install both PyTorch & TF. and I disagree, a user should only need one of them